### PR TITLE
Expand clickable area that leads to subviews

### PR DESCRIPTION
**Pull request by MariagraziaAlastra**
**Reviewer:** @laurengarcia 

## Description:
Header, tracker networks, privacy practices, and all trackers lines should now be entirely clickable.
Previously only clicking on the arrow or the link had any effect.
Also changed the cursor in those areas to indicate they're clickable.

## Steps to test this PR:
1. Build and reload extension
2. Visit any site
3. All the elements listed in the description should be clickable, showing the cursor pointer
4. Popup switches correctly between main view and subviews

## Automated tests:
- [ ] Unit tests
- [ ] Integration tests

###### Reviewer Checklist:
- [ ] **Ensure the PR solves the problem**
- [ ] **Review every line of code**
- [ ] **Ensure the PR does no harm by testing the changes thoroughly**
- [ ] **Get help if you're uncomfortable with any of the above!**
- [ ] Determine if there are any quick wins that improve the implementation

###### PR Author Checklist:
- [ ] Get advice or leverage existing code
- [ ] Agree on technical approach with reviewer (if the changes are nuanced)
- [ ] Ensure that there is a testing strategy (and documented non-automated tests)
- [ ] Ensure there is a documented monitoring strategy (if necessary)
- [ ] Consider systems implications

### CODE REVIEW

File: shared/js/ui/templates/shared/hero.es6.js [hunk 1]
```diff
@@ -15,16 +15,11 @@ module.exports = function (ops) {
...
-  return bel`<a href="#" class="hero__close js-sliding-subview-close">
-    <span class="icon icon__arrow icon__arrow--large icon__arrow--left">
-    </span>
-  </a>`
-}
-
-function renderOpenButton () {
-  return bel`<a href="#" class="hero__open js-hero-open">
-    <span class="icon icon__arrow icon__arrow--large">
+function renderOpenOrCloseButton (isCloseButton) {
```
Comment (MariagraziaAlastra) [inline shared/js/ui/templates/shared/hero.es6.js]: I thought I would simplify and use just one function here

File: shared/js/ui/templates/top-blocked-truncated.es6.js [hunk 2]
```diff
@@ -11,8 +11,8 @@ module.exports = function () {
       <ol class="default-list top-blocked__list top-blocked__list--truncated">
         ${listItems(this.model.companyListMap)}
       </ol>
-      <div class="top-blocked__see-all border--top">
-        <a href="#" class="link-secondary js-top-blocked-see-all">
+      <div class="top-blocked__see-all border--top js-top-blocked-see-all">
+        <a href="javascript:void(0)" class="link-secondary">
```
Comment (MariagraziaAlastra) [inline shared/js/ui/templates/top-blocked-truncated.es6.js]: Chrome complains about javascript:void(0): 
screen shot 2018-01-17 at 1 00 45 pm

Comment (laurengarcia) [reply shared/js/ui/templates/top-blocked-truncated.es6.js]: Oh no -- CSPs. Ugh. We'll have to sync with Jason on this re: Safari.

Comment (laurengarcia) [reply shared/js/ui/templates/top-blocked-truncated.es6.js]: Lets just leave it as `#` for now, sorry for the confusion.

**Comment by laurengarcia** [comment]
Sorry this one slipped thru the cracks, i tried to resolve conflicts just now but will actually defer to you on that.

**Comment by laurengarcia** [comment]
(Obviously I'm good with merging this once conflicts are resolved)

**Comment by MariagraziaAlastra** [comment]
@laurengarcia Conflicts solved 👍

**Pull request merged**